projects/angular/src/utils/i18n/common-strings.interface.ts

```diff
@@ -146,6 +146,14 @@ export interface ClrCommonStrings {
    * Datagrid column resize handler string
    */
   columnSeparatorDescription: string;
+  /**
+   * Datagrid numeric filter from handler string
+   */
+  datagridNumericFilterFrom: string;
+  /**
+   * Datagrid numeric filter to handler string
+   */
+  datagridNumericFilterTo: string;
```

Comment:
Consider using the following naming convention. `numericFilterLabelFrom` and `numericFilterLabelTo`.

```suggestion
  numericFilterLabelFrom: string;
  /**
   * Datagrid numeric filter to handler string
   */
  numericFilterLabelTo: string;
```